docs/manual-EN/5.appendix/sql-ngql.md

```diff
@@ -0,0 +1,108 @@
+# Comparison Between MySQL and nGQL
+
+## Conceptual Comparisons
+
+|Name               | SQL | nGQL          |
+| --- | --- | --- |
+| vertex      | \  | vertex        |
+| edge | \    | edge          |
+| vertex type        | \   | tag           |
+| edge type          | \  | edge type     |
+| vertex identifier          | \ | vid           |
+| edge identifier        | edge id generated by default   | src, dst, rank  |
+| column | column | \ |
+| row | row | \ |
+
+## Basic Data Operations
+
+Name                    | SQL                   | nGQL
+-------------------------| ------------------------ | -----------
+Create (graph) database              | CREATE DATABASE `<database_name>`                    | CREATE SPACE `<space_name>`
+Show (graph) database              | SHOW DATABASES | SHOW SPACES
+Use (graph) database  | USE `<database_name>` | USE `<space_name>`
+Create tag | \ | CREATE TAG `<tag_name>`
+Create edge | \ | CREATE EDGE `<edge_name>`
+Crete table | CREATE TABLE `<tbl_name>` (create_definition,...) | \
+Show columns | SHOW COLUMNS FROM `<tbl_name>` | \
+Show tags/edges | \ | SHOW TAGS \| EDGES
+Describe tags/edges | \ | DESCRIBE TAG \| EDGE `<tag_name | edge_name>`
+Insert data | INSERT INTO `<tbl_name>` [(col_name [, col_name] ...)] {VALUES \| VALUE} [(value_list) [, (value_list)] | INSERT VERTEX `<tag_name>` (prop_name_list[, prop_name_list]) {VALUES \| VALUE} vid: (prop_value_list[, prop_value_list]) <br/> INSERT EDGE `<edge_name>` ( `<prop_name_list>` ) VALUES \| VALUE `<src_vid>` -> `<dst_vid>`[`@<rank>`] : ( `<prop_value_list>` )
```

Comment:
insert in sql is very different from ngql.
sql accepts duplicates records (unless pk or unique key is specified) -- maybe insert ignore in sql is much alike insert in ngql
ngql does not accept duplicate vertices.

Reply:
but I think nebula allows inserting duplicate vertices because the old data is covered with the new one

Reply:
> but I think nebula allows inserting duplicate vertices because the old data is covered with the new one

ngql insert != sql insert     THAT IS CONFUSING TO THOSE DBA. and should be mentioned.
ngql insert == sql `insert ignore`.